app/src/main/res/layout/activity_about.xml

```diff
@@ -86,10 +86,98 @@
                     android:id="@+id/code_for_vilnius_logo"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:paddingBottom="32dp"
+                    android:paddingBottom="8dp"
                     android:src="@drawable/ic_c4v" />
 
                 <TextView
+                    android:id="@+id/thanks_to"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:paddingBottom="8dp"
+                    android:text="@string/thanks_to"
+                    android:textColor="@color/black_87" />
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:orientation="horizontal"
+                    android:paddingBottom="8dp"
+                    android:gravity="center_horizontal">
+
+                    <TextView
+                        android:id="@+id/ineta"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="@string/ineta"
+                        android:textColor="@color/colorPrimary" />
+
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="@string/comma"
```

Comment:
Not a big problem, but all those TextViews could be replaced with one TextView and multiple spannablestring links. See here: https://developer.android.com/reference/android/text/util/Linkify.html